mx.sulong/mx_sulong.py

```diff
@@ -573,10 +573,9 @@ def runLLVMTestCases(args=None):
 
 def runTruffleTestCases(args=None):
     """runs the Sulong test suite"""
-    ensureLLVMBinariesExist()
-    ensureDragonEggExists()
```

Comment:
We still need to call `ensureDragonEggExists()` somewhere, but maybe we should move it to `tools.py` `GCCCompiler.run`.